src/Markdig/Extensions/JiraLinks/JiraLinkInlineParser.cs

```diff
@@ -41,7 +41,7 @@ public override bool Match(InlineProcessor processor, ref StringSlice slice)
             var endKey = slice.Start;
 
             //read as many uppercase characters as required - project key
```

Comment:
```suggestion
            // read as many uppercase characters or digits as required - project key
```

Reply:
Hi, I see everything is added for the test en verified. 
I've added a test to check if the first char in the key is not a digit as this is not allowed by Jira.